Makefile

```diff
@@ -414,9 +414,9 @@ dpdk.clean:
 ebpf.build:
 ifeq ($(WITH_EBPF), true)
 ifeq ($(WITH_EBPF_DOCKER_BUILDER), true)
-	$(MAKE) -C probe/ebpf docker-ebpf-build
+	$(MAKE) -C probe/ebpf clean docker-ebpf-build check
```

Comment:
Why is the `clean` required ?

Reply:
If I read correctly, the `check` uses `sudo`. I'm not sure it's a good idea to require root privileges during the build process

Reply:
clean is used because (for example) if you compile once in normal and after docker-ebpf-build you the target will be done the first time, do docker build will not be done.
It's a specific case, but may we should remove the docker build

Reply:
yes I used sudo because the checker use raw packets socket
not really ideal but you will win lots of time with the checker if you can compile, load and run the ebpf code (sudo is need only for running)
I will do it in 2 pass (compile and test target)

Reply:
This is also the case for the protobuf and easyjson autogenerated files. The problem fixed by using docker was much more severe : the module was successfully compiled but failed to load, even on a clean environment.